src/plugins/kibana_react/public/exit_full_screen_button/exit_full_screen_button.tsx

```diff
@@ -24,6 +24,7 @@ import { EuiIcon, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
 
 export interface ExitFullScreenButtonProps {
   onExitFullScreenMode: () => void;
+  toggleChrome?: boolean;
```

Comment:
Just noticed another thing, @ajwild - the `toggleChrome` flag is only necessary in the dashboard integration, we shouldn't make it part of the `ExitFullScreenButton` interface. Please but this into the dashboard plugin: https://github.com/elastic/kibana/pull/58435/files#diff-42d3b4a54aaaa4907d311e7020d85682R193

```
const ExitFullScreenButton: React.FC<ExitFullScreenButtonProps & { toggleChrome: boolean; }> = props => {
```